src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/TeamQueries.cs

```diff
@@ -82,23 +76,8 @@ public Team Execute(FindByCaptainIdCriteria criteria)
             return _unitOfWork.Context.Teams.Where(t => t.CaptainId == criteria.CaptainId).Select(GetTeamMapping()).SingleOrDefault();
         }
 
-        /// <summary>
-        /// Find teams by given criteria
-        /// </summary>
-        /// <param name="criteria">Search criteria</param>
-        /// <returns>List of <see cref="Team"/>.</returns>
-        public List<Team> Execute(FindByTournamentIdCriteriaOld criteria)
```

Comment:
Why this was removed?

Reply:
becouse it is obsolete and name Old. it no used anymore

Reply:
Are you sure it is not used?

Reply:
was marked with obsolete attribute, and has no references